src/core/ConfigManager.cpp

```diff
@@ -425,6 +441,11 @@ void ConfigManager::loadConfigFile()
 		QDir().mkpath( userProjectsDir() );
 		QDir().mkpath( userSamplesDir() );
 		QDir().mkpath( userPresetsDir() );
+		QDir().mkpath( userGigDir() );
+		QDir().mkpath( userSf2Dir() );
+		QDir().mkpath( vstDir() );
```

Comment:
The directories are only created once, so there will be a user Lmms/plugin/vst folder created when the the user lmms folder in initially created.

If/ when the user sets it to another location ie ```` Program Files/Steinberg/```` no attempt is made to create or write to that location.